src/transport/PointToPointBroker.cpp

```diff
@@ -502,16 +518,73 @@ std::set<int> PointToPointBroker::getIdxsRegisteredForGroup(int groupId)
     return groupIdIdxsMap[groupId];
 }
 
+void PointToPointBroker::initSequenceCounters(int groupId)
+{
+    if (currentGroupId != NO_CURRENT_GROUP_ID) {
+        SPDLOG_WARN("Changing the current group Id in PTP broker ({} -> {})",
+                    currentGroupId,
+                    groupId);
+    }
+    currentGroupId = groupId;
+    int groupSize = getIdxsRegisteredForGroup(groupId).size();
+    // We assume that group indexes will always go from 0 to `groupSize - 1`
+    // We also want to start counting from one. This way, an unset sequence
+    // number (0 by default in protobuf) will indicate that it is unset.
```

Comment:
"This way, an unset sequence number (0 by default in protobuf) will indicate that it is unset." this seems like a tautology. When do we need to distinguish between unset values and set values? It seems like the `orderMsg` bool should tell us whether we should pay attention to sequence numbers.

Reply:
You are right, the comment and check were left behind from a previous version of the PR.